Makefile

```diff
@@ -21,7 +21,7 @@ jenkins-test: build deps license_check linters
 	TEST_PACKAGES=github.com/onosproject/onos-config/... ./build/build-tools/build/jenkins/make-unit
 
 helmit-config: integration-test-namespace # @HELP run helmit gnmi tests locally
-	helmit test -n test ./cmd/onos-config-tests --suite config
+	helmit test -n test ./cmd/onos-config-tests --suite config -c ../../onos-helm-charts
```

Comment:
I think for integration tests we use the published charts. Any reason we should use local charts in the make file?

Reply:
my bad, didn't intend to commit that